lib/dl_file_uploader_lib/dl_file_uploader_lib/redis_model/models/models.py

```diff
@@ -102,6 +102,16 @@ def get_secret_keys(self) -> set[DataKey]:
         return {DataKey(parts=("refresh_token",))}
 
 
+@attr.s(init=True, kw_only=True)
+class YaDocumentsUserSourceProperties(UserSourceProperties):
+    file_type: FileType = attr.ib(default=FileType.yadocuments)
+
+    private_path: Optional[str] = attr.ib(default=None)
+    public_link: Optional[str] = attr.ib(default=None)
+
+    oauth_token: Optional[str] = attr.ib(default=None)
```

Comment:
This is a sensitive field and it needs some additional processing, see the class above